flashlight/app/asr/CMakeLists.txt

```diff
@@ -50,8 +50,8 @@ install(TARGETS fl_asr_decode RUNTIME DESTINATION ${FL_INSTALL_BIN_DIR})
 add_executable(fl_asr_tutorial_inference_ctc ${CMAKE_CURRENT_LIST_DIR}/tutorial/InferenceCTC.cpp)
 add_executable(fl_asr_tutorial_finetune_ctc ${CMAKE_CURRENT_LIST_DIR}/tutorial/FinetuneCTC.cpp)
 
-target_link_libraries(fl_asr_tutorial_inference_ctc fl_pkg_speech ${CMAKE_DL_LIBS})
-target_link_libraries(fl_asr_tutorial_finetune_ctc fl_pkg_speech ${CMAKE_DL_LIBS})
+target_link_libraries(fl_asr_tutorial_inference_ctc fl_pkg_speech fl_pkg_runtime ${CMAKE_DL_LIBS})
```

Comment:
Note to self: I wonder if `CMAKE_DL_LIBS` is still needed/if we can make those linker flags public/inherited properties from a target that builds plugin